engines/saga/detection_tables.h

```diff
@@ -1050,6 +1054,9 @@ static const SAGAGameDescription gameDescriptions[] = {
 		PATCHLIST_ITE,
 		INTROLIST_ITE_AMIGA_GERMAN_AGA,
 		{},
+		{0x53a8, 3730},
```

Comment:
I am thinking, what if instead of bloating all the detection entries, you introduce GFs for these versions and keep this information in a separate table?